.changeset/hip-seahorses-wash.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-api": minor
+---
+
+Validate filename length for uploads to DAM or FileUploads
+
+The filename can't exceed 255 characters.
```

Comment:
255 was chosen because it's also the max length on MacOS and Windows. So a filename longer than 255 characters was probably changed in JS for malicious reasons.